patchwork-model-loader/src/main/resources/patchwork-model-loader.mixins.json

```
@@ -0,0 +1,13 @@
+{
+  "required": true,
+  "package": "net.patchworkmc.mixin.modelloader",
+  "compatibilityLevel": "JAVA_8",
+  "client": [
+    "MixinBakedModelManager",
+    "MixinModelLoader"
+  ],
+  "injectors": {
+    "defaultRequire": 1,
+    "maxShiftBy": 2
```

This doesn't seem to be needed. 
```suggestion
    "defaultRequire": 1
```